resources/config.json

```diff
@@ -26,8 +26,15 @@
 		"photo_multiplier" : 1,
 		"ioniz_multiplier" : 1,
 		"epair_multiplier" : 1,
+        "ioniz": "IonizBetheBlochRossi",
+        "epair": "EpairKelnerKokoulinPetrukhin",
 		"brems" : "BremsKelnerKokoulinPetrukhin",
 		"photo" : "PhotoAbramowiczLevinLevyMaor97",
+        "annihilation": "none",
```

Comment:
None is the default value. Why set it explicit?

Reply:
To show, what possibilities the user has.